tests/unit/components/HdLink.spec.js

```diff
@@ -32,9 +32,10 @@ describe('HdLink', () => {
 
   it('should render <router-link> on :to', () => {
     const to = 'www.homeday.de';
-
-    wrapper.setProps({
-      to,
+    const wrapper = wrapperBuilder({
+      props: {
+        to,
+      },
     });
 
     expect(wrapper.is('router-link')).toBe(true);
```

Comment:
The test passes but it is logging this warning : `[Vue warn]: Unknown custom element: <router-link>`
I would suggest to make use of  `RouterLinkStub` (https://github.com/vuejs/vue-test-utils/pull/366) to have a warning free test 😂✅
```
import { RouterLinkStub } from '@vue/test-utils';

...

it('should render <router-link> on :to', () => {
    const to = 'www.homeday.de';
    const wrapper = wrapperBuilder({
      props: {
        to,
      },
      stubs: {
        RouterLink: RouterLinkStub,
      },
    });
    expect(wrapper.find(RouterLinkStub).props().to).toBe(to);
    expect(wrapper.html()).toMatchSnapshot();
  });
```

Reply:
Yeah, I saw this one. Tried the stub but it didn't work. Gonna try it once again. Thanks for checking it.

I will also merge your latest tests here to keep it in sync =]